components/pluralrules/src/lib.rs

```diff
@@ -128,17 +132,16 @@ pub enum PluralRuleType {
 /// ```
 /// use icu_locale::LanguageIdentifier;
 /// use icu_pluralrules::{PluralRules, PluralRuleType, PluralCategory};
-/// use icu_pluralrules::data::provider::DummyDataProvider;
+/// use icu_data_provider::InvariantDataProvider;
 ///
 /// let lang: LanguageIdentifier = "en".parse()
 ///     .expect("Failed to parse a language identifier.");
 ///
-/// let dp = DummyDataProvider::default();
+/// let dp = InvariantDataProvider;
 ///
 /// let pr = PluralRules::try_new(lang, PluralRuleType::Cardinal, &dp)
 ///     .expect("Failed to construct a PluralRules struct.");
 ///
-/// assert_eq!(pr.select(1_usize), PluralCategory::One);
```

Comment:
Optional: We could make InvariantDataProvider match `1` to plural category one, so that an example like this will work.  However, that might introduce English bias into InvariantDataProvider.

If you want to make this change, edit the definition of `impl Default for PluralRuleStringsV1` in *structs/plurals.rs*

Reply:
I'm ok with not englifying Invariant one :)